# **PROJETO ANÁLISE ESTATÍSTICA DO ENEM 2022**

Este projeto analisa os resultados do ENEM no estado do Espírito Santo do ano de 2022.

Os dados foram extraídos do site do INEP:

https://www.gov.br/inep/pt-br/acesso-a-informacao/dados-abertos/microdados/enem

In [1]:
import numpy as np
import pandas as pd

In [2]:
dados = pd.read_csv('microdados_enem_2022_es.csv', sep=';', encoding='iso-8859-1')
# encoding: codificação de caracteres, normalmente utiliza-se o iso-8859-1, utf-8, latin-1)

## Extração e Limpeza dos dados

In [3]:
pd.options.display.max_columns = 39

In [4]:
dados

NU_INSCRICAO  TP_FAIXA_ETARIA TP_SEXO  TP_ESTADO_CIVIL  TP_COR_RACA  \
0         2,10E+11                6       M                1            3   
1         2,10E+11                4       F                1            3   
2         2,10E+11               12       M                3            2   
3         2,10E+11                2       F                1            3   
4         2,10E+11                7       M                1            0   
...            ...              ...     ...              ...          ...   
18713     2,10E+11                2       F                1            3   
18714     2,10E+11               13       F                2            3   
18715     2,10E+11                2       F                1            3   
18716     2,10E+11                4       F                1            3   
18717     2,10E+11                3       M                1            3   

       TP_NACIONALIDADE  TP_ST_CONCLUSAO  TP_ANO_CONCLUIU  TP_ESCOLA  \
0                     1                1                2          1   
1                     1                1                1          1   
2                     1                1               15          1   
3                     1                2                0          3   
4                     1                1                2          1   
...                 ...              ...              ...        ...   
18713                 1                2                0          2   
18714                 1                1               16          1   
18715                 1                3                0          1   
18716                 1                1                2          1   
18717                 1                2                0          2   

       TP_ENSINO  IN_TREINEIRO  CO_MUNICIPIO_ESC NO_MUNICIPIO_ESC  CO_UF_ESC  \
0            NaN             0               NaN              NaN        NaN   
1            NaN             0               NaN              NaN        NaN   
2            NaN             0               NaN              NaN        NaN   
3            1.0             0               NaN              NaN        NaN   
4            NaN             0               NaN              NaN        NaN   
...          ...           ...               ...              ...        ...   
18713        1.0             0         3205002.0            Serra       32.0   
18714        NaN             0               NaN              NaN        NaN   
18715        NaN             1               NaN              NaN        NaN   
18716        NaN             0               NaN              NaN        NaN   
18717        1.0             0         3205309.0          Vitória       32.0   

      SG_UF_ESC  CO_MUNICIPIO_PROVA NO_MUNICIPIO_PROVA  CO_UF_PROVA  \
0           NaN             3201308          Cariacica           32   
1           NaN             3201506           Colatina           32   
2           NaN             3205002              Serra           32   
3           NaN             3201506           Colatina           32   
4           NaN             3205309            Vitória           32   
...         ...                 ...                ...          ...   
18713        ES             3205002              Serra           32   
18714       NaN             3205200         Vila Velha           32   
18715       NaN             3202801         Itapemirim           32   
18716       NaN             3205200         Vila Velha           32   
18717        ES             3205309            Vitória           32   

      SG_UF_PROVA  TP_DEPENDENCIA_ADM_ESC  TP_LOCALIZACAO_ESC  \
0              ES                     NaN                 NaN   
1              ES                     NaN                 NaN   
2              ES                     NaN                 NaN   
3              ES                     NaN                 NaN   
4              ES                     NaN                 NaN   
...         

In [5]:
#O número total de inscritos em São Paulo foi de 18.717, sendo de 1,78% de todo o Brasil que foi de 1.048.576 inscritos. 
dados.shape

(18718, 39)

In [6]:
# Análise dos tipos de atributos.
# object: strings
# int64: inteiros
# float64: reais
# complex: complexos
dados.dtypes

NU_INSCRICAO               object
TP_FAIXA_ETARIA             int64
TP_SEXO                    object
TP_ESTADO_CIVIL             int64
TP_COR_RACA                 int64
TP_NACIONALIDADE            int64
TP_ST_CONCLUSAO             int64
TP_ANO_CONCLUIU             int64
TP_ESCOLA                   int64
TP_ENSINO                 float64
IN_TREINEIRO                int64
CO_MUNICIPIO_ESC          float64
NO_MUNICIPIO_ESC           object
CO_UF_ESC                 float64
SG_UF_ESC                  object
CO_MUNICIPIO_PROVA          int64
NO_MUNICIPIO_PROVA         object
CO_UF_PROVA                 int64
SG_UF_PROVA                object
TP_DEPENDENCIA_ADM_ESC    float64
TP_LOCALIZACAO_ESC        float64
TP_SIT_FUNC_ESC           float64
TP_PRESENCA_CN              int64
TP_PRESENCA_CH              int64
TP_PRESENCA_LC              int64
TP_PRESENCA_MT              int64
NU_NOTA_CN                float64
NU_NOTA_CH                float64
NU_NOTA_LC                float64
NU_NOTA_MT    

In [7]:
# como boa prática é recomendável criar uma nova variável
dados1 = dados.drop(columns=['CO_MUNICIPIO_ESC'])

In [8]:
dados1 = dados1.drop(columns=['CO_UF_ESC','SG_UF_ESC','CO_MUNICIPIO_PROVA', 'CO_UF_PROVA', 'SG_UF_PROVA'])

In [9]:
dados1 = dados1.drop(columns=['TP_ANO_CONCLUIU','TP_ENSINO','TP_DEPENDENCIA_ADM_ESC','TP_LOCALIZACAO_ESC','TP_SIT_FUNC_ESC'])

In [10]:
dados1.shape

(18718, 28)

In [11]:
dados1.head(2)

NU_INSCRICAO  TP_FAIXA_ETARIA TP_SEXO  TP_ESTADO_CIVIL  TP_COR_RACA  \
0     2,10E+11                6       M                1            3   
1     2,10E+11                4       F                1            3   

   TP_NACIONALIDADE  TP_ST_CONCLUSAO  TP_ESCOLA  IN_TREINEIRO  \
0                 1                1          1             0   
1                 1                1          1             0   

  NO_MUNICIPIO_ESC NO_MUNICIPIO_PROVA  TP_PRESENCA_CN  TP_PRESENCA_CH  \
0              NaN          Cariacica               1               1   
1              NaN           Colatina               1               1   

   TP_PRESENCA_LC  TP_PRESENCA_MT  NU_NOTA_CN  NU_NOTA_CH  NU_NOTA_LC  \
0               1               1       490.7       388.6       357.8   
1               1               1       522.3       473.5       448.5   

   NU_NOTA_MT  TP_LINGUA  TP_STATUS_REDACAO  NU_NOTA_COMP1  NU_NOTA_COMP2  \
0       416.0          1                1.0          120.0           40.0   
1       460.8          1                1.0          100.0          140.0   

   NU_NOTA_COMP3  NU_NOTA_COMP4  NU_NOTA_COMP5  NU_NOTA_REDACAO Q025  
0           40.0          100.0           20.0            320.0    B  
1          100.0          140.0          120.0            600.0    B

Renomeando Variáveis (colunas)

In [12]:
dados1 = dados1.rename(columns={'NU_NOTA_REDACAO': 'NOTA_REDACAO','NU_NOTA_CN': 'NOTA_CN','NU_NOTA_CH': 'NOTA_CH',
                                'NU_NOTA_LC': 'NOTA_LC', 'NU_NOTA_MT': 'NOTA_MT', 'NU_NOTA_COMP1': 'COMP1', 
                                'NU_NOTA_COMP2': 'COMP2', 'NU_NOTA_COMP3': 'COMP3','NU_NOTA_COMP4': 'COMP4',
                                'NU_NOTA_COMP5':'COMP5', 'TP_SEXO': 'SEXO', 'TP_COR_RACA': 'RACA', 
                                'Q025': 'INTERNET', 'TP_ESCOLA':'ESCOLA', 'TP_FAIXA_ETARIA': 'FAIXA_ETARIA',
                               'TP_ESTADO_CIVIL': 'ESTADO_CIVIL', 'TP_NACIONALIDADE': 'NACIONALIDADE'})

Renomeando Registros (Linhas)

In [13]:
dados1["RACA"] = dados1["RACA"].replace({0:"nao_declarado", 1:"branca", 2:"preta", 3:"parda", 4:"amarela", 5:"indigena"})

In [14]:
# OUTRA MANEIRA
# dados1['RACA'] = dados1['RACA'].replace([0,1,2,3,4,5], ['nao_declarado','branca','preta','parda','amarela','indigena'])

In [15]:
dados1["TP_LINGUA"] = dados1["TP_LINGUA"].replace({0:"ingles", 1:"espanhol"})

In [16]:
dados1["ESCOLA"] = dados1["ESCOLA"].replace({1:"nao_respondeu", 2:"publica", 3:"privada"})

In [17]:
dados1["INTERNET"] = dados1["INTERNET"].replace({"A":"nao", "B":"sim"})

In [18]:
dados1['FAIXA_ETARIA'] = dados1['FAIXA_ETARIA'].replace([1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20], 
                                                        ['<_17','17','18','19','20','21','22','23','24','25',
                                                        '26_30','31_35','36_40','41_45','46_50','51_55','56_60',
                                                        '61_65','66_70','>_70'])

In [19]:
dados1.head(2)

NU_INSCRICAO FAIXA_ETARIA SEXO  ESTADO_CIVIL   RACA  NACIONALIDADE  \
0     2,10E+11           21    M             1  parda              1   
1     2,10E+11           19    F             1  parda              1   

   TP_ST_CONCLUSAO         ESCOLA  IN_TREINEIRO NO_MUNICIPIO_ESC  \
0                1  nao_respondeu             0              NaN   
1                1  nao_respondeu             0              NaN   

  NO_MUNICIPIO_PROVA  TP_PRESENCA_CN  TP_PRESENCA_CH  TP_PRESENCA_LC  \
0          Cariacica               1               1               1   
1           Colatina               1               1               1   

   TP_PRESENCA_MT  NOTA_CN  NOTA_CH  NOTA_LC  NOTA_MT TP_LINGUA  \
0               1    490.7    388.6    357.8    416.0  espanhol   
1               1    522.3    473.5    448.5    460.8  espanhol   

   TP_STATUS_REDACAO  COMP1  COMP2  COMP3  COMP4  COMP5  NOTA_REDACAO INTERNET  
0                1.0  120.0   40.0   40.0  100.0   20.0         320.0      sim  
1                1.0  100.0  140.0  100.0  140.0  120.0         600.0      sim

## Análise Exploratória e Transformação dos dados

**Análise das faixa etárias**

In [20]:
dados1['FAIXA_ETARIA'].value_counts()


18       5587
17       2985
19       2636
20       1490
26_30     982
21        918
22        652
<_17      628
31_35     575
23        552
24        424
36_40     392
25        316
41_45     259
46_50     153
51_55      91
56_60      52
61_65      17
66_70       6
>_70        3
Name: FAIXA_ETARIA, dtype: int64

In [21]:
dados1['FAIXA_ETARIA'].value_counts().sort_index()

17       2985
18       5587
19       2636
20       1490
21        918
22        652
23        552
24        424
25        316
26_30     982
31_35     575
36_40     392
41_45     259
46_50     153
51_55      91
56_60      52
61_65      17
66_70       6
<_17      628
>_70        3
Name: FAIXA_ETARIA, dtype: int64

In [22]:
# configura o Pandas para observar XX linhas
#pd.set_option('display.max_rows', XX)

In [23]:
# Para ver todas as linhas, primeiro transforme num Dataframe:
faixa_etaria = pd.DataFrame(dados1['FAIXA_ETARIA'].value_counts().sort_index())
faixa_etaria

FAIXA_ETARIA
17             2985
18             5587
19             2636
20             1490
21              918
22              652
23              552
24              424
25              316
26_30           982
31_35           575
36_40           392
41_45           259
46_50           153
51_55            91
56_60            52
61_65            17
66_70             6
<_17            628
>_70              3

**Treineiros**

In [24]:
dados1.query('IN_TREINEIRO == 1')['IN_TREINEIRO'].value_counts()

1    1572
Name: IN_TREINEIRO, dtype: int64

In [25]:
treineiros = dados1.loc[dados1.IN_TREINEIRO == 1]
treineiros.head(2)

NU_INSCRICAO FAIXA_ETARIA SEXO  ESTADO_CIVIL   RACA  NACIONALIDADE  \
17     2,10E+11         <_17    F             1  preta              1   
26     2,10E+11         <_17    M             1  parda              1   

    TP_ST_CONCLUSAO         ESCOLA  IN_TREINEIRO NO_MUNICIPIO_ESC  \
17                3  nao_respondeu             1              NaN   
26                3  nao_respondeu             1              NaN   

   NO_MUNICIPIO_PROVA  TP_PRESENCA_CN  TP_PRESENCA_CH  TP_PRESENCA_LC  \
17           Linhares               1               1               1   
26       Nova Venécia               1               1               1   

    TP_PRESENCA_MT  NOTA_CN  NOTA_CH  NOTA_LC  NOTA_MT TP_LINGUA  \
17               1    435.6    569.0    562.2    698.8    ingles   
26               1    453.7    543.9    512.7    427.6    ingles   

    TP_STATUS_REDACAO  COMP1  COMP2  COMP3  COMP4  COMP5  NOTA_REDACAO  \
17                1.0  160.0  160.0  160.0  180.0  180.0         840.0   
26                1.0  120.0  200.0  140.0  120.0  140.0         720.0   

   INTERNET  
17      sim  
26      sim

In [26]:
treineiros.to_csv('treineiros_enem_2022_es.csv', encoding = 'iso-8859-1', index = False)

In [27]:
# Separação entre os treineiros e os vestibulandos
vestibulandos = dados1.loc[dados1.IN_TREINEIRO == 0]
vestibulandos.head(2)

NU_INSCRICAO FAIXA_ETARIA SEXO  ESTADO_CIVIL   RACA  NACIONALIDADE  \
0     2,10E+11           21    M             1  parda              1   
1     2,10E+11           19    F             1  parda              1   

   TP_ST_CONCLUSAO         ESCOLA  IN_TREINEIRO NO_MUNICIPIO_ESC  \
0                1  nao_respondeu             0              NaN   
1                1  nao_respondeu             0              NaN   

  NO_MUNICIPIO_PROVA  TP_PRESENCA_CN  TP_PRESENCA_CH  TP_PRESENCA_LC  \
0          Cariacica               1               1               1   
1           Colatina               1               1               1   

   TP_PRESENCA_MT  NOTA_CN  NOTA_CH  NOTA_LC  NOTA_MT TP_LINGUA  \
0               1    490.7    388.6    357.8    416.0  espanhol   
1               1    522.3    473.5    448.5    460.8  espanhol   

   TP_STATUS_REDACAO  COMP1  COMP2  COMP3  COMP4  COMP5  NOTA_REDACAO INTERNET  
0                1.0  120.0   40.0   40.0  100.0   20.0         320.0      sim  
1                1.0  100.0  140.0  100.0  140.0  120.0         600.0      sim

In [28]:
vestibulandos.shape

(17146, 28)

**Análise das presenças**

obs: ver dicionário de dados

In [29]:
vestibulandos['TP_PRESENCA_CN'].value_counts()

1    11839
0     5292
2       15
Name: TP_PRESENCA_CN, dtype: int64

In [30]:
vestibulandos['TP_PRESENCA_CH'].value_counts()

1    12495
0     4623
2       28
Name: TP_PRESENCA_CH, dtype: int64

In [31]:
vestibulandos['TP_PRESENCA_LC'].value_counts()

1    12495
0     4623
2       28
Name: TP_PRESENCA_LC, dtype: int64

In [32]:
vestibulandos['TP_PRESENCA_MT'].value_counts()

1    11839
0     5292
2       15
Name: TP_PRESENCA_MT, dtype: int64

_______________________________________________________________________
RESUMO DA PRESENÇA NAS PROVAS:

Ciências da Natureza: 11.839 presentes, 5.292 faltaram e 15 eliminados.

Ciências Humanas: 12.495 presentes, 4.623 faltaram e 28 eliminados.

Linguagens e Códigos: 12.495 presentes, 4.623 faltaram e 28 eliminados.

Matemática: 11.839 presentes, 5.292 faltaram e 15 eliminados.
_______________________________________________________________________

In [33]:
vestibulandos['TP_STATUS_REDACAO'].value_counts().sort_index()

1.0    11999
2.0        6
3.0      103
4.0      230
6.0      104
7.0        6
8.0       39
9.0        8
Name: TP_STATUS_REDACAO, dtype: int64

**Valores Ausentes**

In [34]:
vestibulandos['FAIXA_ETARIA'].isnull().sum()

0

In [35]:
provas = ['NOTA_REDACAO','NOTA_CN','NOTA_MT','NOTA_LC','NOTA_CH']

In [36]:
# Registros NAN nas notas
vestibulandos[provas].isnull().sum()

NOTA_REDACAO    4651
NOTA_CN         5307
NOTA_MT         5307
NOTA_LC         4651
NOTA_CH         4651
dtype: int64

In [37]:
vestibulandos_classificados = vestibulandos.loc[vestibulandos.TP_PRESENCA_CH == 1]
vestibulandos_classificados = vestibulandos_classificados.loc[vestibulandos_classificados.TP_PRESENCA_MT == 1]

In [38]:
vestibulandos_classificados['TP_PRESENCA_MT'].value_counts()

1    11817
Name: TP_PRESENCA_MT, dtype: int64

In [39]:
vestibulandos_classificados['TP_PRESENCA_CH'].value_counts()

1    11817
Name: TP_PRESENCA_CH, dtype: int64

In [40]:
vestibulandos_classificados['TP_PRESENCA_CN'].value_counts()

1    11817
Name: TP_PRESENCA_CN, dtype: int64

In [41]:
vestibulandos_classificados['TP_PRESENCA_LC'].value_counts()

1    11817
Name: TP_PRESENCA_LC, dtype: int64

In [42]:
vestibulandos_classificados[provas].isnull().sum()

NOTA_REDACAO    0
NOTA_CN         0
NOTA_MT         0
NOTA_LC         0
NOTA_CH         0
dtype: int64

** Notas zeros na Redação**

In [43]:
from collections import Counter
dict(Counter(vestibulandos_classificados['NOTA_REDACAO']))

{320.0: 71,
 600.0: 691,
 780.0: 338,
 580.0: 570,
 660.0: 437,
 620.0: 542,
 720.0: 425,
 540.0: 440,
 500.0: 255,
 640.0: 591,
 480.0: 266,
 400.0: 101,
 700.0: 383,
 560.0: 515,
 940.0: 335,
 680.0: 538,
 820.0: 337,
 520.0: 388,
 840.0: 383,
 440.0: 145,
 860.0: 367,
 800.0: 372,
 760.0: 360,
 380.0: 74,
 880.0: 414,
 340.0: 97,
 420.0: 93,
 920.0: 457,
 460.0: 163,
 900.0: 444,
 740.0: 328,
 360.0: 174,
 0.0: 361,
 300.0: 53,
 960.0: 197,
 280.0: 25,
 980.0: 56,
 260.0: 16,
 240.0: 7,
 200.0: 3,
 180.0: 1,
 40.0: 1,
 220.0: 3}

In [44]:
vestibulandos_classificados.query('NOTA_REDACAO == 0')['NOTA_REDACAO'].value_counts()

0.0    361
Name: NOTA_REDACAO, dtype: int64

In [45]:
vestibulandos_classificados.query('COMP1 == 0')['COMP1'].value_counts()

0.0    362
Name: COMP1, dtype: int64

In [46]:
vestibulandos_classificados.query('COMP2 == 0')['COMP2'].value_counts()

0.0    361
Name: COMP2, dtype: int64

In [47]:
vestibulandos_classificados.query('COMP3 == 0')['COMP3'].value_counts()

0.0    362
Name: COMP3, dtype: int64

In [48]:
vestibulandos_classificados.query('COMP4 == 0')['COMP4'].value_counts()

0.0    362
Name: COMP4, dtype: int64

In [49]:
vestibulandos_classificados.query('COMP5 == 0')['COMP5'].value_counts()

0.0    767
Name: COMP5, dtype: int64

_______________________________________________________________________
NOTAS ZEROS EM REDAÇÃO:

REDAÇÃO (final): 361 notas zeros devido a não cumprimento das regras ou tirou zero em todos os cinco componentes.

COMPONENTE 1(Demonstrar domínio da modalidade escrita formal da Língua Portuguesa.): 362 notas zeros somente neste item.

COMPONENTE 2 (Compreender a proposta de redação e aplicar conceitos das várias áreas de conhecimento para desenvolver o tema, dentro dos limites estruturais do texto dissertativo-argumentativo em prosa): 361 notas zero somente neste item. 

COMPONENTE 3 (Selecionar, relacionar, organizar e interpretar informações, fatos, opiniões e argumentos em defesa de um ponto de vista): 362 notas zeros somente neste item.

COMPONENTE 4 (Demonstrar conhecimento dos mecanismos linguísticos necessários para a construção da argumentação.): 362 notas zeros somente neste item.

COMPONENTE 5 (Elaborar proposta de intervenção para o problema abordado, respeitando os direitos humanos): 767 notas zeros somente neste item.
_______________________________________________________________________

In [50]:
vestibulandos_classificados.query('NOTA_CH == 0')['NOTA_CH'].value_counts()

0.0    9
Name: NOTA_CH, dtype: int64

In [51]:
vestibulandos_classificados.query('NOTA_CN == 0')['NOTA_CN'].value_counts()

0.0    1
Name: NOTA_CN, dtype: int64

In [52]:
vestibulandos_classificados.query('NOTA_LC == 0')['NOTA_LC'].value_counts()

0.0    2
Name: NOTA_LC, dtype: int64

In [53]:
vestibulandos_classificados.query('NOTA_MT == 0')['NOTA_MT'].value_counts()

0.0    2
Name: NOTA_MT, dtype: int64

In [54]:
vestibulandos_classificados.query('NOTA_REDACAO == 0')['NOTA_REDACAO'].value_counts()

0.0    361
Name: NOTA_REDACAO, dtype: int64

_______________________________________________________________________
NOTAS ZEROS DOS INSCRITOS CONCORRENTES AO VESTIBULAR:

Ciências Humanas = 9

Ciências da Natureza = 1

Linguagens e Códigos = 2

Matemática = 2

Redação = 361
_______________________________________________________________________

In [55]:
vestibulandos_classificados.head()

NU_INSCRICAO FAIXA_ETARIA SEXO  ESTADO_CIVIL   RACA  NACIONALIDADE  \
0      2,10E+11           21    M             1  parda              1   
1      2,10E+11           19    F             1  parda              1   
3      2,10E+11           17    F             1  parda              1   
9      2,10E+11        41_45    F             3  parda              1   
11     2,10E+11           17    M             1  parda              1   

    TP_ST_CONCLUSAO         ESCOLA  IN_TREINEIRO NO_MUNICIPIO_ESC  \
0                 1  nao_respondeu             0              NaN   
1                 1  nao_respondeu             0              NaN   
3                 2        privada             0              NaN   
9                 1  nao_respondeu             0              NaN   
11                2        publica             0            Viana   

   NO_MUNICIPIO_PROVA  TP_PRESENCA_CN  TP_PRESENCA_CH  TP_PRESENCA_LC  \
0           Cariacica               1               1               1   
1            Colatina               1               1               1   
3            Colatina               1               1               1   
9          Vila Velha               1               1               1   
11          Cariacica               1               1               1   

    TP_PRESENCA_MT  NOTA_CN  NOTA_CH  NOTA_LC  NOTA_MT TP_LINGUA  \
0                1    490.7    388.6    357.8    416.0  espanhol   
1                1    522.3    473.5    448.5    460.8  espanhol   
3                1    455.0    450.0    401.5    449.3  espanhol   
9                1    383.7    439.0    399.1    437.6  espanhol   
11               1    407.6    542.1    525.4    464.2    ingles   

    TP_STATUS_REDACAO  COMP1  COMP2  COMP3  COMP4  COMP5  NOTA_REDACAO  \
0                 1.0  120.0   40.0   40.0  100.0   20.0         320.0   
1                 1.0  100.0  140.0  100.0  140.0  120.0         600.0   
3                 1.0  160.0  160.0  140.0  160.0  160.0         780.0   
9                 1.0  120.0  120.0  120.0  120.0  100.0         580.0   
11                1.0  120.0  120.0  120.0  160.0  140.0         660.0   

   INTERNET  
0       sim  
1       sim  
3       sim  
9       sim  
11      nao

In [56]:
vestibulandos_classificados = vestibulandos_classificados.drop(columns=['IN_TREINEIRO'], )

In [57]:
vestibulandos_classificados.shape

(11817, 27)

## **Salvando (Exportando) o Dataframe Tratado**

In [58]:
vestibulandos_classificados.to_csv('enem_2022_es_tratado.csv', encoding = 'iso-8859-1', index = False)